### PR TITLE
Update runtime to 24.08

### PR DESCRIPTION
**Pull request by ColinKinloch**

### CODE REVIEW

**Comment by rafaelmardojai** [comment]
Runtime `22.08` is now EOL, any update on merging this?

**Comment by hadess** [comment]
> Runtime `22.08` is now EOL, any update on merging this?

Have you tested the build mentioned above? If so, it would be useful to mention it.

**Comment by hadess** [comment]
bot, build

**Comment by hadess** [comment]
@ColinKinloch can you update it to `24.08` so we can spend the next 2 years without warnings?

**Comment by ColinKinloch** [comment]
oops

**Comment by ColinKinloch** [comment]
bot, build

File: com.adobe.Flash-Player-Projector.appdata.xml [hunk 1]
```diff
@@ -4,6 +4,9 @@
   <id>com.adobe.Flash-Player-Projector</id>
   <name>Adobe Flash Player</name>
   <summary>Player for content created using Adobe Flash</summary>
+  <developer id="com.adobe">
```
Comment (hadess) [inline com.adobe.Flash-Player-Projector.appdata.xml:7]: You should remove that bit, it's already there 2 lines below (added in https://github.com/flathub/com.adobe.Flash-Player-Projector/pull/71)

File: com.adobe.Flash-Player-Projector.appdata.xml [hunk 2]
```diff
@@ -22,7 +25,7 @@
     </p>
     <p>NOTE: This wrapper is not verified by, affiliated with, or supported by Adobe, and does not contain a web browser plug-in.</p>
   </description>
-  <url type="homepage">https://www.adobe.com/support/flashplayer/debug_downloads.html</url>
+  <url type="homepage">https://www.adobe.com/products/flashplayer/end-of-life-alternative.html</url>
```
Comment (hadess) [inline com.adobe.Flash-Player-Projector.appdata.xml:28]: Can you please make this a separate commit?

**Comment by ColinKinloch** [comment]
bot, build

**Comment by hadess** [comment]
Great, thank you both for your help!
